README.md

```diff
@@ -1,3 +1,78 @@
-# Teams
+Teams
+=====
 
-This repo to hold teams! File applications for teams and leadership as issues here.
+Aspects
+-------
+
+__Team Leader:__ Stefan Zimmermann
+
+The Aspects team is responsible for implementing the Aspects defined in
+cEP-0005.
+
+
+cib
+---
+
+__Team Leader:__ Adrian Zatreanu
+
+The cib team develops and maintains the cib tool for coala.
+
+
+Community
+---------
+
+__Team Leader:__ Maximilian Scholz
+(max@coala.io)
+
+The community team organizes community bonding events like meetups and
+gaming evenings to support and promote a healthy, happy community.
+It also serves as a contact for personal problems or conflicts between
+community members.
+
+
+Core
+----
+
+__Team Leader:__ Adhityaa Chandrasekar
+
+Developing and maintaining the coala core repository.
+In addition everything that does not fit into another team's domain,
+will be assigned to the core team.
+
+
+Docs
+----
+
+__Team Leader:__ Adrian Zatreanu
+
+Improve the existing documentation, wiki, tutorials and guides.
+Introduce new ones as needed.
+
+
+Research
+--------
+
+__Team Leader:__ Udayan Tandon
+
+Work on new architecture for coala parts. Help with thesis writing and
+implementing algorithms for the science community.
+
+
+Applications
+============
+
+Team Leader Appplication
+------------------------
+
+To apply for a team lead position, you have to create an issue in this repository using the given template.
+
+
+Team Member Application
+-----------------------
+
+To apply to any team, you just have to message the team leader in some way.
+For example via gitter, an issue in this repository or with an email.
```

Comment:
jup. Though we need both, the team lead and the team member application in there
or maybe just make it more generic so it can be used for both